### ISSUE TITLE
Green lightning bug

### ISSUE THREAD

**Issue by llacczy** [elysium-project/server#107]
There is sometimes some weird lightning going on in ironforge, i think it is because of a quest item
(i dont know wich, but there are some pictures)

Green lightning in ironforge:
http://imgur.com/a/0Urls

Possible bug source:
http://imgur.com/a/Ml0UV

(i hope this report is helpful by any means, since i dont know what is that quest item)

**Comment by brotalnia** [comment]
Did you ask other people if they are seeing them too? Could be a graphics card issue on your side.

**Comment by llacczy** [comment]
Yes, once after raid we got back to if and multiple ppl were able to see the green lightning.
Also, i asked once again in guild chat and got confirmed so this is happening to more ppl.

Thanks for the reply.

**Comment by llacczy** [comment]
related item: Egan's Blaster
http://db.vanillagaming.org/?item=13289

Using this item unintentional, like long range on a player, with a wall between causes this green lightning for others, changin the item to be only usable on the intented targets probably will get rid of this unintentional/ exploit behaviour.


related quest:
The Restless Souls
http://db.vanillagaming.org/?quest=5282#wh-comments
